engine/tests/phpunit/Elgg/Amd/ConfigTest.php

```diff
@@ -41,31 +52,39 @@ public function testCanRequireUnregisteredAmdModules() {
      * @expectedException \InvalidParameterException
      */
 	public function testThrowsOnBadShim() {
-		$amdConfig = new \Elgg\Amd\Config();
-		$amdConfig->addShim('bad_shim', array('invalid' => 'config'));
+		$this->amdConfig->addShim('bad_shim', array('invalid' => 'config'));
 
-		$configArray = $amdConfig->getConfig();
+		$configArray = $this->amdConfig->getConfig();
 
 		$this->assertEquals(array('jquery'), $configArray['deps']);
 	}
 
 	public function testCanAddModuleAsAmd() {
-		$amdConfig = new \Elgg\Amd\Config();
-		$amdConfig->addModule('jquery');
+		$this->amdConfig->addModule('jquery');
 
-		$configArray = $amdConfig->getConfig();
+		$configArray = $this->amdConfig->getConfig();
 
 		$this->assertEquals(array('jquery'), $configArray['deps']);
 	}
 
 	public function testCanAddModuleAsShim() {
-		$amdConfig = new \Elgg\Amd\Config();
-		$amdConfig->addModule('jquery.form', array('exports' => 'jquery.fn.ajaxform'));
+		$this->amdConfig->addModule('jquery.form', array('exports' => 'jquery.fn.ajaxform'));
 
-		$configArray = $amdConfig->getConfig();
+		$configArray = $this->amdConfig->getConfig();
 
 		$this->assertArrayHasKey('jquery.form', $configArray['shim']);
 		$this->assertEquals(array('exports' => 'jquery.fn.ajaxform'), $configArray['shim']['jquery.form']);
 	}
-}
 
+	public function testCheckAmdConfigHookTrigger() {
```

Comment:
What is this test checking?

Did you mean "getConfig triggers the 'config, amd' plugin hook"?

If so, I'd write it like that: `testGetConfigTriggersTheConfigAmdPluginHook()`